### ISSUE TITLE
flutter 3.24.3 com.github.jitpack:android-example

### ISSUE THREAD

**Issue by wjh19970625** [flutter/flutter#156409]
### Actual results

Attribute application@theme value=(@style/TransTheme) from [:image_pickers] AndroidManifest.xml:35:9-42
	is also present at [com.github.jitpack:android-example:1.0.1] AndroidManifest.xml:15:9-40 value=(@style/AppTheme).
	Suggestion: add 'tools:replace="android:theme"' to <application> element at AndroidManifest.xml:39:5-96:19 to override.

### Code sample

<details open><summary>Code sample</summary>

```dart
[Paste your code here]
```

</details>

### Screenshots or Video

<details open>
<summary>Screenshots / Video demonstration</summary>

[Upload media here]

</details>

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

**Comment by darshankawar** [comment]
@wjh19970625 
I am not sure if I understood what are you trying to report here. Please elaborate by providing what exactly is the issue are you facing, steps to replicate, runnable code sample, actual vs expected behavior and `flutter doctor -v`.

**Comment by wjh19970625** [comment]
android
flutter 3.19.0
运行项目正常
flutter 3.24.3
出现了异常






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "flutter/flutter"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月9日(星期三) 下午2:51
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [flutter/flutter] flutter 3.24.3 com.github.jitpack:android-example (Issue #156409)





 
@wjh19970625
 I am not sure if I understood what are you trying to report here. Please elaborate by providing what exactly is the issue are you facing, steps to replicate, runnable code sample, actual vs expected behavior and flutter doctor -v.

**Comment by wjh19970625** [comment]
flutter 3.19.0运行正常&nbsp; &nbsp;升级到flutter 3.24.3 后 出现下面问题&nbsp;


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "flutter/flutter"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月9日(星期三) 下午2:51
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [flutter/flutter] flutter 3.24.3 com.github.jitpack:android-example (Issue #156409)





 
@wjh19970625
 I am not sure if I understood what are you trying to report here. Please elaborate by providing what exactly is the issue are you facing, steps to replicate, runnable code sample, actual vs expected behavior and flutter doctor -v.

**Comment by wjh19970625** [comment]
抱歉 发了很多没用的邮箱 第一次使用 按回车就会发送 我的本意是换行



---Original---
From: ***@***.***&gt;
Date: Wed, Oct 9, 2024 14:51 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [flutter/flutter] flutter 3.24.3com.github.jitpack:android-example (Issue #156409)




 
@wjh19970625
 I am not sure if I understood what are you trying to report here. Please elaborate by providing what exactly is the issue are you facing, steps to replicate, runnable code sample, actual vs expected behavior and flutter doctor -v.

**Comment by darshankawar** [comment]
@wjh19970625 
I am finding it hard to follow what you are trying to say. Please post in English for better readability and provide a working code sample that we can use and provide actual vs expected result.

**Comment by matt3o** [comment]
I ran into the same issue and was able to pin it down to a single package: flutter_sound: ^9.0.0
Creating a new project and just adding this one dependency will trigger the error for me. The recommended fix creates even weirder problems and is not the solution.

`dependencies:
  flutter_sound: ^9.0.0
`

In a new package under Windows for me this trigger the exception above, I'll paste mine for documentation.


```Launching lib\main.dart on sdk gphone64 x86 64 in debug mode...
C:\Users\Hadlich\code\flutter_bug\flutter_application_1\android\app\src\debug\AndroidManifest.xml Error:
	Attribute application@icon value=(@mipmap/ic_launcher) from (unknown)
	is also present at [com.github.jitpack:android-example:1.0.1] AndroidManifest.xml:13:9-45 value=(@drawable/ic_launcher).
	Suggestion: add 'tools:replace="android:icon"' to <application> element at AndroidManifest.xml to override.
C:\Users\Hadlich\code\flutter_bug\flutter_application_1\android\app\src\debug\AndroidManifest.xml Error:
	Attribute application@label value=(flutter_application_1) from (unknown)
	is also present at [com.github.jitpack:android-example:1.0.1] AndroidManifest.xml:14:9-41 value=(@string/app_name).
	Suggestion: add 'tools:replace="android:label"' to <application> element at AndroidManifest.xml to override.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugMainManifest'.
> Manifest merger failed with multiple errors, see logs

* Try:
```

```[√] Flutter (Channel stable, 3.24.3, on Microsoft Windows [Version 10.0.19045.5011], locale de-DE)
    • Flutter version 3.24.3 on channel stable at C:\Users\Hadlich\.flutter\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 2663184aa7 (6 weeks ago), 2024-09-11 16:27:48 -0500
    • Engine revision 36335019a8
    • Dart version 3.5.3
    • DevTools version 2.37.3

[√] Windows Version (Installed version of Windows is version 10 or higher)

[√] Android toolchain - develop for Android devices (Android SDK version 34.0.0)     
    • Android SDK at C:\Users\Hadlich\AppData\Local\Android\Sdk
    • Platform android-34, build-tools 34.0.0
    • ANDROID_HOME = C:\Users\Hadlich\AppData\Local\Android\Sdk
    • Java binary at: C:\Program Files\Android\Android Studio\jbr\bin\java
    • Java version OpenJDK Runtime Environment (build 17.0.10+0--11572160)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop Windows apps (Visual Studio Build Tools 2022 17.8.6)     
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools
    • Visual Studio Build Tools 2022 version 17.8.34525.116
    • Windows 10 SDK version 10.0.22621.0

[√] Android Studio (version 2023.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.10+0--11572160)

[√] VS Code (version 1.94.2)
    • VS Code at C:\Users\Hadlich\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.98.0

[√] Connected device (4 available)
    • sdk gphone64 x86 64 (mobile) • emulator-5554 • android-x64    • Android 14 (API 34) (emulator)
    • Windows (desktop)            • windows       • windows-x64    • Microsoft Windows [Version 10.0.19045.5011]
    • Chrome (web)                 • chrome        • web-javascript • Google Chrome 129.0.6668.103
    • Edge (web)                   • edge          • web-javascript • Microsoft Edge 130.0.2849.46

[√] Network resources
    • All expected network resources are available.

• No issues found!```

**Comment by bradintheusa** [comment]
I'm getting the same issue when using `package: flutter_sound: ^9.0.0`

**Comment by wjh19970625** [comment]
thank you,I'll try



---Original---
From: ***@***.***&gt;
Date: Wed, Oct 30, 2024 23:23 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [flutter/flutter] flutter 3.24.3com.github.jitpack:android-example (Issue #156409)




 
I'm getting the same issue when using package: flutter_sound: ^9.0.0

**Comment by Rawkstar618** [comment]
@darshankawar Were you able to find a fix or a package to replace flutter_sound? I am facing the same issue and the project delivery is soon.

**Comment by mohammadparhizkary** [comment]
hi, go to your android manifest file , then try adding "xmlns:tools="http://schemas.android.com/tools"" in your manifest tag on top of your android manifest  : eg: <manifest xmlns:android="http://schemas.android.com/apk/res/android"
        xmlns:tools="http://schemas.android.com/tools"> , now in your application tag , add "tools:replace="android:icon, android:label"", hope this works for you

**Comment by bradintheusa** [comment]
I did this but now my icon is just blank.

**Comment by drunkenQCat** [comment]
I'm using  flutter_sound 9.16.3, and met this problem, too. Wired but the only way to solve it now is what @mohammadparhizkary mentioned.  Maybe it's better to open an issue on flutter_sound's repo @wjh19970625.

**Comment by drunkenQCat** [comment]
@bradintheusa @mohammadparhizkary @Rawkstar618 @matt3o the problem of flutter_sound has been solved by the developer. just upgrade the package, and restore the AndroidManifest.xml, everything will be fine.

**Comment by bradintheusa** [comment]
Everything runs fine but my icon is still blank. I'll assume that's an issue just for me if everyone else ok?

**Comment by drunkenQCat** [comment]
> Everything runs fine but my icon is still blank. I'll assume that's an issue just for me if everyone else ok?

no, my icon was also in blank after upgrade. but after clean up some files in `android` folder, it works. maybe you can have a look on [my commit](https://github.com/drunkenQCat/voislate/commit/2bcef30be3f1cb41c5bbc2c21e6f91edc5c89c31)

**Comment by shishi-RN** [comment]
use flutter_sound: ^9.17.0
